static-build/qemu/Dockerfile

```diff
@@ -39,7 +40,7 @@ RUN git clone  https://github.com/qemu/keycodemapdb.git ui/keycodemapdb
 
 ADD configure-hypervisor.sh /root/configure-hypervisor.sh
 
-RUN /root/configure-hypervisor.sh -s kata-qemu | xargs ./configure --prefix=/opt/kata --with-pkgversion=kata-static
+RUN if [ "$goarch" = "amd64" ]; then /root/configure-hypervisor.sh -s kata-qemu | xargs ./configure --prefix=/opt/kata --with-pkgversion=kata-static; else ./configure --prefix=/opt/kata --with-pkgversion=kata-static ; fi
```

Comment:
why don't use `configure-hypervisor.sh` ?

Reply:
@devimc :  I see compilation issues when I use `configure-hypervisor.sh` on ppc64le.
1. Options like `--disable-uuid`and ` --disable-fdt ` need  not be for ppc64le. i.e. we cannot disable it on ppc64le.

2. After enabling the above options, I still still see  the following. Need to look into it:
For your reference:
```
Building spapr-rtas/spapr-rtas.img
  CC      qga/commands.o
  CC      qga/guest-agent-command-state.o
  CC      qga/main.o
  CC      qga/commands-posix.o
  Building spapr-rtas/spapr-rtas.bin
  CC      qga/channel-posix.o
  CC      qga/qapi-generated/qga-qapi-types.o
rm spapr-rtas.img spapr-rtas.o
  CC      qga/qapi-generated/qga-qapi-visit.o
  CC      qga/qapi-generated/qga-qmp-marshal.o
  AR      libqemuutil.a
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(dl-sysdep.o): warning: multiple common of `__cache_line_size'
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(libc-start.o): warning: previous common is here
  LINK    fsdev/virtfs-proxy-helper
  LINK    scsi/qemu-pr-helper
  LINK    qemu-bridge-helper
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x2c4): warning: Using 'getpwuid' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x168): warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x1ac): warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(dl-sysdep.o): warning: multiple common of `__cache_line_size'
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(libc-start.o): warning: previous common is here
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libcap-ng.a(cap-ng.o): In function `capng_change_id':
(.text+0x2f1c): warning: Using 'initgroups' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x2c4): warning: Using 'getpwuid' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x168): warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x1ac): warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(dl-sysdep.o): warning: multiple common of `__cache_line_size'
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(libc-start.o): warning: previous common is here
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libcap-ng.a(cap-ng.o): In function `capng_change_id':
(.text+0x2f1c): warning: Using 'initgroups' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
scsi/qemu-pr-helper.o: In function `main':
/root/qemu/scsi/qemu-pr-helper.c:975: warning: Using 'getgrnam' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
scsi/qemu-pr-helper.o: In function `prh_co_entry':
/root/qemu/scsi/qemu-pr-helper.c:745: warning: Using 'getpwnam' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x2c4): warning: Using 'getpwuid' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x168): warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libglib-2.0.a(libglib_2_0_la-gutils.o): In function `g_get_user_database_entry':
(.text+0x1ac): warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
io/dns-resolver.o: In function `qio_dns_resolver_lookup_sync_inet':
/root/qemu/io/dns-resolver.c:88: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(dl-sysdep.o): warning: multiple common of `__cache_line_size'
/usr/lib/gcc/powerpc64le-linux-gnu/5/../../../powerpc64le-linux-gnu/libc.a(libc-start.o): warning: previous common is here
```

Reply:
those warnings are because of you're building a static qemu `-s`, I see the same warnings on x86_64

Reply:
@devimc : FWIW its not only warnings but an ultimate failure as well due too unknown/unused variables. 

Complete Logs: https://paste.fedoraproject.org/paste/UWlBXwJKs0e28V08fr9uVw

Reply:
ohh I see, I guess those errors are ppc specific

Reply:
I still think we need to fix `configure-hypervisor.sh` for ppc64le since a default configure is going to created a qemu binary with "too many" features enabled which will impact memory density / boot speed.

Reply:
Ah - maybe this PR is blocked on #94 :)